Client/Frontend/Widgets/Menu/MenuViewController.swift

```diff
@@ -160,7 +160,17 @@ class MenuViewController: UIViewController {
 
     override func viewWillLayoutSubviews() {
         super.viewWillLayoutSubviews()
+        relayoutMenuAndSetSize()
+    }
+
+    override func viewWillAppear(animated: Bool) {
+        super.viewWillAppear(animated)
+        relayoutMenuAndSetSize()
```

Comment:
I am a little concerned that this will cause relayoutMenuAndSetSize to get called more than once at times and that will affect performance, but as I cannot figure out how to deploy to iOS10 right now I can't test it to support or disprove the concern.